msrestazure/azure_active_directory.py

```diff
@@ -525,3 +531,32 @@ def set_token(self, response_url):
             raise_with_traceback(AuthenticationError, "", err)
         else:
             self.token = token
+
+class AdalAuthentication(Authentication):#pylint: disable=too-few-public-methods
```

Comment:
Probably a docstring that explains what the class is about. Same goes for methods.